Source/WebKit/WebProcess/Plugins/PDF/WKAccessibilityPDFDocumentObject.mm

```diff
@@ -231,12 +273,10 @@ - (void)gotoDestination:(PDFDestination *)destination
     if (!plugin)
         return;
 
-    NSInteger pageIndex = [_pdfDocument indexForPage:[destination page]];
-    NSRect rect = [[self accessibilityChildren][0] accessibilityFrame];
-    WebCore::IntPoint point = { 0, static_cast<int>(rect.size.height * pageIndex) + 1 };
+    int pageIndex = (int) [_pdfDocument indexForPage:[destination page]];
 
-    callOnMainRunLoopAndWait([protectedPlugin = plugin, &point] {
-        protectedPlugin->setScrollOffset(point);
+    callOnMainRunLoop([protectedPlugin = plugin, &pageIndex] {
```

Comment:
Now that you're no longer calling `AndWait()`, you can no longer capture stuff by reference. It isn't safe.
`&pageIndex` -> `pageIndex`
also, `protectedPlugin = plugin` -> `plugin`. No reason for the more complicated code.